### DISCUSSION TITLE
How to change the boot parameters (Jetson AGX Xavier DevKit)

### DISCUSSION THREAD

**Discussion by FabioSpano** [OE4T/meta-tegra#928] [Q&A] [answered]
Hi everyone!

I successfully did the boot on my Jetson AGX Xavier Development Kit with 32GB. When I do `cat /proc/cmdline` I see that in the boot parameters memory is set to 5000M (`MEM=5000M`). In order to make my Jailhouse configuration work I need to set MEM to 8000M, but I could not find where to set the boot parameters. 

So here is my question: where can I change these parameters? 

Thanks!

**Comment by madisongh** [comment, marked as answer]
You can add kernel command line parameters with the KERNEL_ARGS variable, so something like
```
KERNEL_ARGS:jetson-agx-xavier-devkit:append = " mem=8000M"
```
in your `local.conf` would append that setting.

But if you're seeing an explicit `mem=5000M` now, I'm not sure where that's coming from.  There's nothing in this layer that sets that; perhaps something in one of the other layers in your build is doing so?  You might want to track that down first.

**Reply by FabioSpano** [reply]
Thank you!
